pom.xml

```diff
@@ -33,6 +33,7 @@
   </scm>
 
   <properties>
+    <revision>2.4.6-SNAPSHOT</revision>
```

Comment:
Without context, `revision` is ambiguous. Suggest renaming to something more descriptive (e.g., `kcl.version`).

Reply:
In https://maven.apache.org/maven-ci-friendly.html#multi-module-setup, it mentioned: `A note about the used properties. You can only use those named ${revision}, ${sha1} and/or ${changelist} and not other named properties like this` 
Other names will be prohibited.

Reply:
That limitation appears to be from `flatten-maven-plugin` [interpolation](https://github.com/mojohaus/flatten-maven-plugin/blob/c613968d1fe7b748945e4e232ecbe66739610feb/src/main/java/org/codehaus/mojo/flatten/cifriendly/CiModelInterpolator.java#L149-L153) -- a plugin we do not use today. Other property names appear to work.

LGTM either way!